### ISSUE TITLE
2.0.7 not building with null safety

### ISSUE THREAD

**Issue by steveneal** [the-airbender/quill_html_editor#23]
I get the following error when trying to build the latest version:

`Error: Method 'insert' cannot be called on 'OverlayState?' because it is potentially null.`

FIX:

el_tooltip.dart line 150:

 ```
   if (_overlayEntryHidden != null) {
      overlayStateHidden?.insert(_overlayEntryHidden!); // need the ? operator here
    }

```

**Comment by the-airbender** [comment]
Hi @steveneal 

Thanks for reporting. Will fix and release a new version today.

**Comment by the-airbender** [comment]
Hi @steveneal,

I am in Flutter Ver 3.7.3 and able to build. Could you please share few more details ?

1. Flutter and Dart version
2. Steps to recreate
3. Platforms - Android or iOS or Web

Thanks!

**Comment by the-airbender** [comment]
Hi @steveneal , 

Closing the issue as  I am unable to recreate this. Kindly reopen this with more info, if you still face any issues with the newer version. 

Thanks

**Comment by steveneal** [comment]
Hi this is still a problem in 2.1.3.

Errors when compiling my project appear in el_tooltop.dart lines:150 and 246:

`../../.pub-cache/hosted/pub.dartlang.org/quill_html_editor-2.1.3/lib/src/widgets/el_tooltip/el_tooltip.dart:150:26: Error: Method 'insert' cannot be called on 'OverlayState?' because it is potentially null.`

`../../.pub-cache/hosted/pub.dartlang.org/quill_html_editor-2.1.3/lib/src/widgets/el_tooltip/el_tooltip.dart:246:20: Error: Method 'insert' cannot be called on 'OverlayState?' because it is potentially null`

I can edit the files locally (adding a ? before the dot operator) to get it working but this isn't ideal. 

Output from Flutter doctor:

`[✓] Flutter (Channel stable, 3.3.9, on macOS 13.0.1 22A400 darwin-arm, locale en-GB)`
`[✓] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)`
`[✓] Xcode - develop for iOS and macOS (Xcode 14.1)`
`[✓] Chrome - develop for the web`
`[✓] Android Studio (version 2022.1)`
`[✓] IntelliJ IDEA Ultimate Edition (version 2022.1)`
`[✓] VS Code (version 1.75.1)`
`[✓] Connected device (4 available)`
`[✓] HTTP Host Availability`
`• No issues found!`

**Comment by the-airbender** [comment]
Hi @steveneal,

Thank you for the Flutter doctor output. I have tested it in latest version of flutter 3.7.5,  it seems the error is coming in the older versions. I will downgrade the version, will give a fix that compatible for all versions.

Thanks

**Comment by the-airbender** [comment]
Hi @steveneal 

Turns out, the `Overlay.of(context)` has become a non nullable on the latest flutter sdk versions. That is the reason for the issue on older versions. 

While this can be resolved by updating flutter to 3.7, I looking for a work around to support older versions. 

Will post the update here. 

Thanks!

**Comment by the-airbender** [comment]
Hi @steveneal,

Fixed the issue in newer version [2.1.4](https://pub.dev/packages/quill_html_editor/versions/2.1.4). Please check it out and share your feedback

Thanks!
